config/networking.yml

```diff
@@ -14,9 +14,9 @@ cfroutesync:
   image: #@ data.values.images.cfroutesync
 
   ccCA: #@ base64.decode(data.values.system_certificate.ca)
-  ccBaseURL: #@ "https://api.{}".format(data.values.system_domain)
+  ccBaseURL: http://capi.cf-system.svc.cluster.local
   uaaCA: #@ base64.decode(data.values.system_certificate.ca)
-  uaaBaseURL: #@ "https://uaa.{}".format(data.values.system_domain)
+  uaaBaseURL: http://uaa.cf-system.svc.cluster.local:8080
```

Comment:
should change this to `#@ "http://uaa." + data.values.system_domain + ".svc.cluster.local:8080"`

Reply:
Oh yeah right, thanks. I adjusted it.